### PR TITLE
Feature/basket mobile

### PR DESCRIPTION
**Pull request by carlwood**
screen shot 2015-09-01 at 14 47 45

screen shot 2015-09-01 at 14 48 03

### CODE REVIEW

File: views/basket/mobile.php [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+                if ($item->variant->shipping->collection_only) {
+
+                    echo '<div class="alert alert-warning alert-mini">';
+                        echo '<strong>Note:</strong> Collection only.';
+                    echo '</div>';
+                }
+     
+                ?>
+
+            <div>
```
Comment (gsdd) [inline views/basket/mobile.php]: Is the div tag here required, also there are 2 closing divs further down (124 and 126) that don't look right. Are you closing one too many?

Comment (gsdd) [inline views/basket/mobile.php]: Ah, maybe this is supposed to be a closing tag?

File: views/basket/mobile.php [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+                    <div class="col-xs-12"><div class="pull-left">Shipping</div>';
+
+                    echo '<div class="pull-right">';
+
+                    if ($shippingType === 'DELIVER') {
+
+                        if (empty($readonly)) {
+
+                            echo '<select id="selectDeliveryOption" class="form-control bump-up">
+                        <option>Standard delivery  - Free</option>
```
Comment (gsdd) [inline views/basket/mobile.php]: Indentation might be a bit off here

File: views/basket/index.php [hunk 3]
```diff
@@ -100,6 +100,8 @@
 
                 $this->load->view($skin->path . 'views/basket/table', $tableData);
 
+                $this->load->view($skin->path . 'views/basket/mobile');
```
Comment (gsdd) [inline views/basket/index.php]: Can you check if we need to pass $tableData in to this view? If not, can you check whether or not it is required for the table view too? Seems like one must be wrong!

File: views/basket/mobile.php (hunk 1 above)
Comment (carlwood) [inline views/basket/mobile.php]: Line 81 is a div that is just used for creating a block and new line. I close this on line 124. (fixed indentation now)

Line 126 is the closing of `col-xs-7` on line 42

**Pull request merged**